airbyte-integrations/connectors/source-hubspot/source_hubspot/streams.py

```diff
@@ -807,6 +855,24 @@ def _process_search(
 
         return list(stream_records.values()), raw_response
 
+    def _read_associations(self, records: Iterable, identifiers: Iterable[Union[int, str]]) -> Iterable[Mapping[str, Any]]:
+        records_by_pk = {record[self.primary_key]: record for record in records}
```

Comment:
Why do we pass both identifiers and records if we can get identifiers from records?

Reply:
good point! fixed, thanks